app/controllers/healthcheck/healthchecks_controller.rb

```diff
@@ -28,5 +30,11 @@ def verbose_error(checker)
     def verbose?
       Healthcheck.configuration.verbose
     end
+
+    def token_valid?
+      return true if Healthcheck.configuration.token.blank?
+
+      Healthcheck.configuration.token == params[:token]
```

Comment:
Sorry man, I forgot to send this comment. 

I just ask about to replace it to:

```suggestion
      Healthcheck.configuration.token.blank? || Healthcheck.configuration.token == params[:token]
```